Dockerfile.amd64

```diff
@@ -0,0 +1,12 @@
+FROM amd64/busybox:1.28.3
+
+LABEL maintainer "Tom Denham <tom@tigera.io>"
+
+ADD dist/amd64/ /opt/cni/bin/
```

Comment:
I'm _slightly_ wary of this sort of change.  Previously, we whitelisted the files that went into the container, now we'll include anything that happens to be in that directory.  The thing that usually causes this to go wrong is when you switch branches without running `make clean` and the list of files in the directory is different on the different branches.

That said, the release target does an explicit clean, so we're fairly safe from _shipping_ unexpected files. I think I can live with it given it's a nice simplification.